### PR TITLE
Initial Widget Inspector page

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: inspector [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Flutter Widget Inspector
+
+The Flutter framework uses Widgets as [the core building
+block](https://flutter.io/widgets-intro/) for anything from controls (text,
+buttons, toggles, etc.) to layout (centering, padding, rows, columns, etc.). The
+Flutter Widget Inspector is powerful tool for visualizing and exploring Flutter
+these Widget trees. It can be helpful when:
+
+* Undestanding existing layouts
+* Determining which source code produces a particular widget
```
Comment (jacob314) [inline inspector]: When using the --preview-dart-2 and --track-creation-location flags.

File: _includes/sidebar.html [hunk 2]
```diff
@@ -49,6 +49,7 @@
       <li><a href="/hot-reload/">Using hot reload</a></li>
       <li><a href="/testing/">Test your app</a></li>
       <li><a href="/debugging/">Debug your app</a></li>
+      <li><a href="/inspector/">Inspect your app widgets</a></li>
```
Comment (sethladd) [inline _includes/sidebar.html]: maybe just "Inspect your UI" ? "app widgets" seems redundant

File: inspector.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* TOC Placeholder
+{:toc}
+
+## Flutter Widget Inspector
+
+The Flutter framework uses Widgets as [the core building
+block](https://flutter.io/widgets-intro/) for anything from controls (text,
+buttons, toggles, etc.) to layout (centering, padding, rows, columns, etc.). The
+Flutter Widget Inspector is powerful tool for visualizing and exploring Flutter
+these Widget trees. It can be helpful when:
```
Comment (sethladd) [inline inspector.md]: do we always capitalize W in Widget?

File: inspector.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+* Undestanding existing layouts
+* Determining which source code produces a particular widget
+* Diagnosing layout issues
+
+For a demonstration of the Widget Inspector, please see the recent
+[DartConf talk](https://www.youtube.com/watch?v=JIcmJNT9DNI).
+
```
Comment (sethladd) [inline inspector.md]: Could we add one screenshot here? To give a flavor of what it does.

File: inspector.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+## Get Started with the Inspector
+
+The Widget Inspector is currently available in [the Flutter
+plugin](https://flutter.io/ide-setup/) for Android Studio, or IntelliJ IDEA.
```
Comment (sethladd) [inline inspector.md]: Screenshot for which button to click to activate inspector?

File: inspector.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+permalink: /inspector/
+---
+
+The Flutter Widget Inspector is powerful tool for visualizing and exploring
+Flutter Widget trees.
+
+* TOC Placeholder
+{:toc}
+
+## Flutter Widget Inspector
```
Comment (sethladd) [inline inspector.md]: At the end of this page, maybe a link for where people ask for help and file bugs.

**Comment by mit-mit** [comment]
@jacob314 can you help with screenshots and steps as suggested by Seth above?

File: inspector (hunk 1 above)
Comment (mit-mit) [reply inspector]: Removed from now; will add back when it's generally part of the Dart 2 preview flag

File: inspector.md (hunk 5 above)
Comment (mit-mit) [reply inspector.md]: In the new M22 build launching this week it auto-activated whenever you run an app!

File: inspector.md (hunk 6 above)
Comment (mit-mit) [reply inspector.md]: done

**Comment by jacob314** [comment]
Go ahead and land and I'll send a pull request with screenshots

**Comment by mit-mit** [comment]
Already did :-)

**Comment by jacob314** [comment]
I'll create a pull request with screenshots.

**Pull request merged**